machine_management/deleting-machine.adoc

```diff
@@ -9,3 +9,8 @@ toc::[]
 You can delete a specific machine.
 
 include::modules/machine-delete.adoc[leveloffset=+1]
+
+[role="_additional-resources"]
+.Additional resources
```

Comment:
@Amrita42, sorry to hold up this short PR, but I just realized that we're using `=` for _Additional resources_ in assemblies, per the contributing guidelines:

> Additionally, in an assembly, use == formatting for the section heading (== Additional resources). Use of this heading syntax at the assembly level indicates that the sections relate to the whole assembly. For example:

https://github.com/openshift/openshift-docs/blob/main/contributing_to_docs/doc_guidelines.adoc#additional-resources-sections

Looks like an easy change, and then we can get this merged, thanks!

Reply:
Thanks @jboxman-rh , Done.

Reply:
@Amrita42, also requires an `id` such as:

```
[id="additional-resources_configuring-alert-notifications"]
```

per the guidelines. Thanks!

Reply:
@jboxman-rh done..